commands/init.go

```diff
@@ -40,6 +40,7 @@ var initCmd = &cmds.Command{
 		cmdkit.BoolOption(DevnetStaging, "when set, populates config bootstrap addrs with the dns multiaddrs of the staging devnet and other staging devnet specific bootstrap parameters."),
 		cmdkit.BoolOption(DevnetNightly, "when set, populates config bootstrap addrs with the dns multiaddrs of the nightly devnet and other nightly devnet specific bootstrap parameters"),
 		cmdkit.BoolOption(DevnetUser, "when set, populates config bootstrap addrs with the dns multiaddrs of the user devnet and other user devnet specific bootstrap parameters"),
+		cmdkit.BoolOption(CatchupSyncTestMode, "when set, the daemon is places in a special mode for chain catchup operations").WithDefault(false),
```

Comment:
nit: "placed"

also what does it mean placed in a "special mode" it would help to be more specific here.

Reply:
forget about the "be more specific" idea as its clear from below and a temporary option

Reply:
I'll will adjust the flag description to make it more clear that this is temporary .